lib/p2p/Pool.ts

```diff
@@ -31,8 +31,8 @@ interface Pool {
   on(event: 'packet.getOrders', listener: (peer: Peer, reqId: string, pairIds: string[]) => void): this;
   on(event: 'packet.orderInvalidation', listener: (orderInvalidation: OrderPortion, peer: string) => void): this;
   on(event: 'peer.close', listener: (peerPubKey?: string) => void): this;
-  /** Adds a listener to be called when a peer has newly advertised pairs. */
-  on(event: 'peer.pairsAdvertised', listener: (peer: Peer, pairIds: string[]) => void): this;
+  /** Adds a listener to be called when a peer's advertised but inactive pairs should be verified. */
```

Comment:
nit: `advertised, but`

Reply:
I think grammatically it's correct either way. To me it reads better without the punctuation, but if this was confusing to you I can change it.